doc/development_process/gh_labels.rst

```diff
@@ -0,0 +1,128 @@
+fix it.
+
+This saves us all time when searching, reduces the chances of the PR or issue
+being forgotten, speeds up reviewing, reduces the changes of duplicates, etc.
+
+These are the labels we currently have, grouped by type:
+
+Area:
+*****
+
+=============  ===============================================================
+Labels         ``area:*``
+Applicable to  PRs  and issues
+Description    Indicates which areas are affected by the bug or pull request
+=============  ===============================================================
+
+An area maintainer should be able to filter by his area and
+find all issues and PRs which relate to that area.
+
+Platform
+*********
+
+=============  ===============================================================
+Labels         ``platform:*``
+Applicable to  PRs  and issues
+Description    An issue or PR which affects only a particular platform
+=============  ===============================================================
+
+To be discussed in a meeting
+****************************
+
+=============  ===============================================================
+Labels         ``API``, ``dev-review``, ``TSC``
+Applicable to  PRs  and issues
+Description    The issue is to be discussed in the following API/dev-review/TSC
+               meeting if time permits.
+=============  ===============================================================
+
+Minimum PR review time
+**********************
+
+=============  ===============================================================
+Labels         ``Hot Fix``, ``Trivial PR``, ``Maintainer``,
+               ``Security Review``, ``TSC``
+Applicable to  PRs only
+Description    Depending on the PR complexity, an indication of how long a merge
+               should be held to ensure proper review. See
+               :ref:`review process <review_time>`
+=============  ===============================================================
+
+Issue priority labels
+*********************
+
+=============  ===============================================================
+Labels         ``priority:{high|medium|low}``
+Applicable to  Issues only
+Description    To classify the priority/importance of a bug or feature
+=============  ===============================================================
+
+Note: Issue priorities are decided/changed during the bug-triage or TSC
+meetings.
+
+Miscellaneous labels
+********************
+
+For both PRs and issues
+=======================
+
++-------------------+----------------------------------------------------------+
+|``Bug``            | The issue is a bug, or the PR is fixing a bug            |
++-------------------+----------------------------------------------------------+
+|``Coverity``       |A Coverity detected issue or its fix                      |
++-------------------+----------------------------------------------------------+
+|Waiting for        | The Zephyr developers are waiting for the submitter to   |
```

Comment:
This label is currently called ``waiting for response``.  This name should also be enclosed in backticks to be formatted like all the others in this table.